core/src/main/java/org/geysermc/geyser/translator/inventory/InventoryTranslator.java

```diff
@@ -216,6 +220,22 @@ public ItemStackResponse translateRequest(GeyserSession session, Inventory inven
                     boolean isSourceCursor = isCursor(transferAction.getSource());
                     boolean isDestCursor = isCursor(transferAction.getDestination());
 
+                    if (destSlot == 5 || sourceSlot == 5) {
```

Comment:
This code probably should be moved specifically to PlayerInventoryTranslator? Because slots move around depending on the inventory, I believe, and this could be accidentally called where it shouldn't be.

Reply:
sure; will give this a shot - I'll hopefully get around to it tomorrow

Reply:
fixed